content/en/synthetics/cicd_integrations/configuration.md

```diff
@@ -0,0 +1,387 @@
+---
+title: CI/CD Integrations Configuration
+kind: documentation
+description: Configure Synthetics to run test in your CI/CD pipelines
+further_reading:
+- link: "/synthetics/cicd_integrations"
+  tag: "Documentation"
+  text: "Learn about Synthetics and CI/CD"
+- link: "/synthetics/ci_results_explorer"
+  tag: "Documentation"
+  text: "Learn about the CI Results Explorer"
+- link: "/synthetics/testing_tunnel"
+  tag: "Documentation"
+  text: "Learn about the Testing Tunnel"
+---
+
+## Overview
+
+Use the `@datadog-ci` NPM package to run Synthetic tests directly within your CI/CD pipeline. You can automatically halt a build, block a deployment, and roll back a deployment when a Synthetics test detects a regression. 
+
+You can configure on which URL your test starts by providing a `startUrl` to your test object and build your own starting URL using any part of your test's original starting URL and the following environment variables:
```

Comment:
Do we want to include this section in the Overview? Or do we want to highlight here `You can customize the Synthetic test configuration to suit your needs within your CI/CD pipeline, check out the configuration options available below`?